.changeset/tricky-shrimps-hunt.md

```diff
@@ -3,55 +3,67 @@
 "@fluidframework/tree": minor
 "__section": feature
 ---
-TreeAlpha identifier APIs for converting, retrieving, and generating identifiers have been added
+New TreeAlpha identifier APIs for converting, retrieving, and generating identifiers
 
 #### TreeAlpha.identifier
+
```

Comment:
Should we enable `markdown-lint` for our changesets? Would catch these sorts of formatting issues. Alternatively, we could just run prettier on them